cumulusci/tasks/salesforce/promote_package_version.py

```diff
@@ -79,6 +86,30 @@ def _run_task(self) -> None:
         target_package_info = self._get_target_package_info(version_id)
         self._promote_package_version(target_package_info)
 
+    def _resolve_version_id(self) -> str:
+        """
+        If a SubscriberPackageVersionId has not been specified we need to
+        auto-resolve a version_id from the tag on our current commit
+
+        @returns: (str) the SubscriberPackageVersionId from the current repo commit
+        """
+        self.logger.info(
+            "No version_id specified. Automatically resolving to latest available Beta version."
+        )
+        tag_name = self.project_config.get_latest_tag(beta=True)
+        repo = self.project_config._get_repo()
+        tag = get_tag_by_name(repo, tag_name)
+
+        for line in tag.message.split("\n"):
+            if line.startswith("version_id:"):
+                version_id = line.split("version_id: ")[1]
+                assert version_id.startswith("04t")
```

Comment:
If it doesn't, then it's clearly not what we're looking for, and maybe we should just ignore it so the user gets the message on line 111 instead of an AssertionError?